sdk/storage/inc/blobs/blob_client.hpp

```diff
@@ -220,7 +209,7 @@ namespace Azure { namespace Storage { namespace Blobs {
      * @param options Optional parameters to execute this function.
      * @return A AbortCopyBlobResponse on successfully aborting.
      */
-    Azure::Core::Response<AbortCopyBlobResponse> AbortCopyFromUri(
+    Azure::Core::Response<AbortCopyBlobInfo> AbortCopyFromUri(
```

Comment:
What about `AbortCopy` API? Are they having the same name?

Reply:
There isn't such an API

Reply:
https://docs.microsoft.com/en-us/rest/api/storageservices/abort-copy-blob

Reply:
There's only one Copy and one Abort Copy API in client API. This aligns with other sdk